app/utils.js

```diff
@@ -83,3 +85,9 @@ exports.sendMailQ = function(toEmail, subject, template, templateParams, attachm
 exports.formatBTCFromSatoshis = function(satoshis) {
   return (satoshis * 1e-8).toFixed(4);
 };
+
+exports.deriveChildKey = function(masterXpub, derivationPath) {
+  const masterNode = prova.HDNode.fromBase58(masterXpub);
```

Comment:
I don't think this is coin agnostic. Also, use a different library like bitgo-utxo-lib and equivalent

Reply:
As discussed over WhatsApp, this is fine for all secp256k1 coins. bitgo-utxo-lib shouldn't be necessary. Also, we use xpubs for all networks, no need to worry about tpubs, etc.